### PR TITLE
added todo flutter app by Mohammad Fayaz

### PR DESCRIPTION
**Pull request by fayaz07**
I have added a sample todo app in flutter, please review this. Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,10 @@ If you appreciate the content 📖, support projects visibility, give 👍| ⭐|
  
   [<img src = "/art/particle.gif" width=240>](http://dartpad.dev/6613d586f958bf692f17b4407fe4e2cb)
 
+- [Todo app](https://gist.github.com/fayaz07/16c9149a3775c6c289625b89d10c62ef) by [Mohammad Fayaz](https://github.com/fayaz07)
```
Comment (divyanshub024) [inline README.md]: Please add a date at the end.

Comment (fayaz07) [reply README.md]: Thank you for reviewing my PR, I have also written Unit, Widget and Integration tests to this, shall I add a link at the end?

File: README.md [hunk 2]
```diff
@@ -126,6 +126,10 @@ If you appreciate the content 📖, support projects visibility, give 👍| ⭐|
 
  [<img src="/art/color_app.gif" width=240>](https://dartpad.dev/cd3b46d3331644c21fe1e62165aa4bac)
 
+- [Todo app](https://gist.github.com/fayaz07/16c9149a3775c6c289625b89d10c62ef) by [Mohammad Fayaz](https://github.com/fayaz07) (30/03/2020)
```
Comment (divyanshub024) [inline README.md]: Please provide the dartpad link not github gist link.

**Comment by divyanshub024** [comment]
@fayaz07 There are the following issues in your code.
```
display1' is deprecated and shouldn't be used. This is the term used in the 2014 version of material design. The modern term is headline4. This feature was deprecated after v1.13.8. - line 85
info
'display1' is deprecated and shouldn't be used. This is the term used in the 2014 version of material design. The modern term is headline4. This feature was deprecated after v1.13.8. - line 90
info
'title' is deprecated and shouldn't be used. This is the term used in the 2014 version of material design. The modern term is headline6. This feature was deprecated after v1.13.8. - line 116
```

**Comment by fayaz07** [comment]
Sorry for that I didn't check the warnings, I have updated the gist. Thank you

File: README.md [hunk 3]
```diff
@@ -130,6 +130,11 @@ If you appreciate the content 📖, support projects visibility, give 👍| ⭐|
 
   [<img src = "/art/buttons.gif" width=240>](https://dartpad.dev/214eee8a820ba4bcff73279a93281a53)
 
+- [Todo app](https://dartpad.dev/16c9149a3775c6c289625b89d10c62ef) by [Mohammad Fayaz](https://github.com/fayaz07) (30/03/2020)
+
+  [<img src = "/art/todo.gif" width=240>](https://dartpad.dev/16c9149a3775c6c289625b89d10c62ef)
+
+
```
Comment (divyanshub024) [inline README.md]: Can you remove this extra line?

**Review by divyanshub024 — APPROVED** [review]
Thanks a lot, for your contribution 🙏

**Comment by fayaz07** [comment]
@divyanshub024 Thank you for accepting my PR

**Pull request merged**